### PR TITLE
Adjusted README and buildscript

### PR DESCRIPTION
**Pull request by pgeier**
Hello,

thanks for providing cegcc, http://cegcc.sourceforge.net/ and svn seem to be dead. Yours seems to be the only repository one can find via google.

Following the build instructions from README.rst and your webpage I get an error like

```
gcc -c -I. -I/home/geier/work/ka/cegcc-build/binutils/binutils -W -Wall -Wstrict-prototypes -Wmissing-prototypes -Wshadow -Wstack-usage=262144 -I/home/geier/work/ka/cegcc-build/binutils/binutils/../zlib -g -O2  /home/geier/work/ka/cegcc-build/binutils/binutils/syslex_wrap.c
/home/geier/work/ka/cegcc-build/binutils/binutils/syslex_wrap.c:25:10: fatal error: syslex.c: No such file or directory
 #include "syslex.c"
          ^~~~~~~~~~

```

By further investigating syslex_wrap.c, a comment gives a hint that something with configuring went wrong:

```
/* Wrapper source to ensure that config.h is the first header file seen by
   the compiler.  */
```

Hence I started working through the build script `build-mingw32ce.sh` where I fortunately found these lines

```
# Don't build in source directory, it will clobber things and cleanup is hard.
if [ -d ${BUILD_DIR}/.svn ]; then
	echo "Don't build in a source directory, please create an empty directory and build there."
	echo "Example :"
	echo "  mkdir build-mingw32ce"
	echo "  cd build-mingw32ce"
	echo "  sh ../build-mingw32ce.sh $@"
	exit 1
fi
```

Accordingly creating a `build-mingw32ce` directory in `build-arm-mingw32ce` and then calling `sh ../build-mingw32ce.sh` did the job.

Nevertheless, I spent some time fixing the problem. Google did not help to much.
Accordingly I adjusted the lines above to check for

```
if [ -d ${BUILD_DIR}/.svn ] || [ -e ${BUILD_DIR}/.gitignore ]; then
```

since it is not svn anymore and furthermore I added a few lines to the README.

Am I getting something wrong? If not you may just accept the pull request and update your personal webpage.

Cheers, Philipp

### CODE REVIEW

File: build-arm-mingw32ce/build-mingw32ce.sh [hunk 1]
```diff
@@ -173,7 +173,7 @@ fi
 components=`echo "${components}" | sed -e 's/[ 	,][ 	,]*/,/g' -e 's/,$//'`
 
 # Don't build in source directory, it will clobber things and cleanup is hard.
-if [ -d ${BUILD_DIR}/.svn ]; then
+if [ -d ${BUILD_DIR}/.svn ] || [ -e ${BUILD_DIR}/.gitignore ]; then
```
Comment (MaxKellermann) [inline build-arm-mingw32ce/build-mingw32ce.sh:179]: The `.svn` check can be removed, we'll never use Subversion again; and this should better do `-d .git` instead of `-e .gitignore`.

File: README.rst [hunk 2]
```diff
@@ -10,7 +10,9 @@ To build::
  git submodule init
  git submodule update
  cd build-arm-mingw32ce
- ./build-mingw32ce.sh --prefix=/where/to/install
+ mkdir build-mingw32ce
+ cd build-mingw32ce
+ sh ../build-mingw32ce.sh --prefix=/where/to/install
```
Comment (MaxKellermann) [inline README.rst]: What's the purpose of this change? I don't understand.

**Comment by MaxKellermann** [comment]
Thanks for the PR; your `.svn` change looks good, and if it were a separate commit, I could easily cherry-pick it. I don't understand the other change, and your text sounds like you don't understand it either. I cannot imagine what problem this could possibly solve.
Formal complaint: if your PR needs text, then this means your commit message is bad. And "Adjusted README and buildscript" is a pretty bad commit message. All it does is state what can easily be seen in `git show --stat`; it doesn't explain at all **what** is adjusted and **why**.

File: README.rst (hunk 2 above)
Comment (pgeier) [reply README.rst]: Calling `./build-mingw32ce.sh` from `build-arm-mingw32ce` directly caused configuring problems for binutil (i.e. the problem with "syslex.c").

Accordingly I tried to follow the steps in the build script `build-mingw32ce.sh` and found the case where for `.svn` is checked and it is suggested to create a build directory and call the build scripts from another build directory by doing `../build-mingw32ce.sh`. Although I now have a build directory inside the build directory, all configuration went right.

Perhaps it's not the optimal solution and I've been to fast with these changes or I should have opened an issue.

I have some other problems with windows header anyway and hence need some put some further effort into this... 
So maybe let's leave it like it is for now until I have the other problems fixed and reworked everything. At least some communication went on.

**Comment by MaxKellermann** [comment]
Huh, I don't see this going anywhere. If you want me to pull something, please create a new PR with small and documented commits.
